Makefile

```diff
@@ -9,9 +9,7 @@ export IMAGE_VERSION = v1.21.0
 export SIMPLE_VERSION = $(shell (test "$(shell git describe --tags)" = "$(shell git describe --tags --abbrev=0)" && echo $(shell git describe --tags)) || echo $(shell git describe --tags --abbrev=0)+git)
 export GIT_VERSION = $(shell git describe --dirty --tags --always)
 export GIT_COMMIT = $(shell git rev-parse HEAD)
-export K8S_VERSION = 1.23
-# TODO: bump this to 1.21, after kubectl `--generator` flag is removed from e2e tests.
-export ENVTEST_K8S_VERSION = 1.23.1
+export K8S_VERSION = 1.24
```

Comment:
Did we check here if the kubebuilder tools 1.24.1 is download after the change?
Note that ENVTEST_K8S_VERSION probably is used here to get the kubebuilder-tolls, see: https://storage.googleapis.com/kubebuilder-tools

That is probably the reason for the CI be falling.
It is a block for the release so could we just revert the changes and do the bumps?

K8S_VERSION = 1.24
export ENVTEST_K8S_VERSION = 1.24.1

c/c @varshaprasad96 @VenkatRamaraju @everettraven

Reply:
Yes, envtest needs a patch version.

Reply:
Modifying `K8S_VERSION` to 1.24.1 should solve the issue.

Reply:
In case it doesn't and if its taking time to debug we could just revert as @camilamacedo86 said, but I'm pretty sure changing it to 1.24.1 would make tests to pass.

Reply:
can be as well as @varshaprasad96 suggested

Reply:
@VenkatRamaraju What @varshaprasad96 is saying checks out. I think we could either set `K8S_VERSION` to `1.24.0` or `1.24.1`

Reply:
K8S_VERSION isn't used by the ENVTEST variable. They are separate AFAIK.

Reply:
Nevermind. I see we are getting rid of the specific ENV_TEST variable